trelloapp.py

```diff
@@ -0,0 +1,124 @@
+import requests
+import os
+from flask import Flask, render_template
+import json
+from dotenv import load_dotenv  #to invoke .env file
+load_dotenv()
+
+trelloapp = Flask(__name__)
+
+#Module2 project #
+#APP_KEY = os.environ.get('APP_KEY')  # take env variable from local system envor
+#APP_TOKEN=os.environ.get('APP_TOKEN')
+
+APP_KEY = os.getenv("APP_KEY")  # take .env from dotenv
+APP_TOKEN = os.getenv("APP_TOKEN")
+
+board_id= '5ef1186ddea5ff1b03e085e6'
+list_id_ToDo = '5ef1186d26a8d939ea575069'
```

Comment:
These 3 list ids could be inferred by name (with the special names documented in the README)